index.html

```diff
@@ -35,55 +35,308 @@
+                or sign-ups required.
+              </p>
+              <div class="languages">
+                <span class="language-used">HTML</span>
+                <span class="language-used">CSS</span>
+                <span class="language-used">javaScript</span>
+              </div>
+              <div class="link">
+                <a href="#" class="project-link">See Project</a>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- fourth project -->
+        <div id="fourth-project" class="project">
+          <div class="project-image">
+            <img
+              src="./Assets/other-images/project-2.jpg"
+              alt="project"
+              class="first-project-image"
+            />
+          </div>
+          <div class="project-content">
+            <h1 class="project-heading">Multi-Post Stories</h1>
+            <div class="project-body">
+              <div class="description">
+                <p class="content-text canopy">CANOPY</p>
+                <img
+                  src="./Assets/vectors/Counter.svg"
+                  class="count-vector"
+                  alt=""
+                />
+                <p class="content-text">BACK END DEV</p>
+                <img
+                  src="./Assets/vectors/Counter.svg"
+                  class="count-vector"
+                  alt=""
+                />
+                <p class="content-text">2015</p>
+              </div>
+              <p class="description-text">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="languages">
+                <span class="language-used">HTML</span>
+                <span class="language-used">CSS</span>
+                <span class="language-used">javaScript</span>
+              </div>
+              <a href="#" class="project-link">
+                <div class="link">See Project</div>
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- about section -->
+      <section id="about-section">
+        <div class="about">
+          <h1 class="about-header">About Myself</h1>
+          <h1 class="about-text">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </h1>
+          <p class="connect">LET’S CONNECT</p>
+          <div class="connect-links">
+            <a href="#">
+              <img
+                src="./Assets/Icons/twitter-vector.svg"
+                alt="twitter"
+                class="social-icons"
+              />
+            </a>
+            <a href="#">
+              <img
+                src="./Assets/Icons/linkedin-vector.svg"
```

Comment:
Because here it's a list of links, please use ul/li tag to have a good semantic web